packages/backend/server/src/plugins/copilot/session.ts

```diff
@@ -347,6 +352,35 @@ export class ChatSessionService {
       );
   }
 
+  async getQuota(userId: string) {
+    const quota = await this.quota.getUserQuota(userId);
```

Comment:
only send quota query if user doesn't have unlimited feature, a db
query can be saved